.github/workflows/release.yml

```diff
@@ -280,10 +282,14 @@ jobs:
         aws-region: us-east-1
 
     - name: Upload package to S3
-      run: aws s3 cp pganalyze-collector-${{ steps.get_version.outputs.version }}.tgz s3://charts.pganalyze.com/pganalyze-collector-${{ steps.get_version.outputs.version }}.tgz
+      run: |
+        aws s3 cp --acl public-read \
+        pganalyze-collector-${{ steps.get_version.outputs.version }}.tgz s3://charts.pganalyze.com/pganalyze-collector-${{ steps.get_version.outputs.version }}.tgz
 
     - name: Create or update index.yaml
       run: helm repo index . --merge index.yaml --url https://charts.pganalyze.com/
 
     - name: Upload index.yaml to S3
-      run: aws s3 cp index.yaml s3://charts.pganalyze.com/index.yaml
+      run: |
+        aws s3 cp --acl public-read --content-type application/yaml --cache-control no-cache \
```

Comment:
Whenever I upload yaml file, the content type `application/octet-stream` is picked up. Let's specify the proper one. Also make sure to add no-cache header as this file can change.

Fun fact: did you know that yaml's content type has officially decided just this year?
https://httptoolkit.com/blog/yaml-media-type-rfc/